examples/motoko_examples/superheroes/dfx.json

```diff
@@ -0,0 +1,20 @@
+{
+  "canisters": {
+    "superheroes": {
+      "type": "custom",
+      "build": "npx azle superheroes",
+      "root": "src",
+      "ts": "src/superheroes/superheroes.ts",
+      "candid": "src/superheroes/superheroes.did",
+      "wasm": "target/wasm32-unknown-unknown/release/superheroes.wasm"
+    }
+  },
+  "www": {
```

Comment:
The www object should be within the canisters object, it is not and thus is not deploying

Reply:
True, seems like this was changed after testing 🤦🏽